packages/blocks/src/api/constants.js

```diff
@@ -127,10 +130,12 @@ export const __EXPERIMENTAL_STYLE_PROPERTY = {
 	fontWeight: {
```

Comment:
Directly above this line, is `fontStyle` missing an opt-in to `useEngine`?

Reply:
Good call! 👍